specification/purview/data-plane/Azure.Analytics.Purview.Scanning/stable/2023-09-01/scanningService.json

```diff
@@ -0,0 +1,10292 @@
+        "domain": {
+          "type": "string",
+          "description": "The domain of the scan."
+        },
+        "workers": {
+          "format": "int32",
+          "type": "integer",
+          "description": "The number of workers of the scan."
+        },
+        "createdAt": {
+          "format": "date-time",
+          "type": "string",
+          "description": "The creation time of the scan.",
+          "readOnly": true
+        },
+        "lastModifiedAt": {
+          "format": "date-time",
+          "type": "string",
+          "description": "The last modified time of the scan.",
+          "readOnly": true
+        },
+        "connectedVia": {
+          "allOf": [
+            {
+              "$ref": "#/definitions/ConnectedVia"
+            }
+          ],
+          "description": "The integration runtime reference of the scan."
+        },
+        "isPresetScan": {
+          "type": "boolean",
+          "description": "The flag to indicate whether the scan is a preset scan or not."
+        },
+        "isLiveViewEnabled": {
+          "type": "boolean",
+          "description": "The flag to indicate whether the live view is enabled or not."
+        },
+        "parallelScanCount": {
+          "format": "int32",
+          "type": "integer",
+          "description": "The number of parallel scans."
+        },
+        "logLevel": {
+          "type": "string",
+          "description": "The log level of the scan."
+        }
+      }
+    },
+    "ExpandingResourceScanProperties": {
+      "type": "object",
+      "description": "expanding resource scan properties",
+      "allOf": [
+        {
+          "$ref": "#/definitions/ScanProperties"
+        }
+      ],
+      "properties": {
+        "resourceTypes": {
+          "type": "object",
+          "properties": {
+            "None": {
+              "$ref": "#/definitions/ResourceTypeFilter"
+            },
+            "AzureSubscription": {
+              "$ref": "#/definitions/ResourceTypeFilter"
+            },
+            "AzureResourceGroup": {
+              "$ref": "#/definitions/ResourceTypeFilter"
+            },
+            "AzureSynapseWorkspace": {
+              "$ref": "#/definitions/ResourceTypeFilter"
+            },
+            "AzureSynapse": {
+              "$ref": "#/definitions/ResourceTypeFilter"
+            },
+            "AdlsGen1": {
+              "$ref": "#/definitions/ResourceTypeFilter"
+            },
+            "AdlsGen2": {
+              "$ref": "#/definitions/ResourceTypeFilter"
+            },
+            "AmazonAccount": {
+              "$ref": "#/definitions/ResourceTypeFilter"
+            },
+            "AmazonS3": {
+              "$ref": "#/definitions/ResourceTypeFilter"
+            },
+            "AmazonSql": {
+              "$ref": "#/definitions/ResourceTypeFilter"
+            },
+            "AzureCosmosDb": {
+              "$ref": "#/definitions/ResourceTypeFilter"
+            },
```

Comment:
What's these?

1st, property name should be camel case.
2nd, I am not sure what it does.

Reply:
This is expanded scan properties for different data source types, for different data source types, they have different definition.
I changed the datasource name to fit in camel case style